FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -210,40 +210,99 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                <locus from="115ra" to="115va"></locus>
+                                <title ref="LIT6353MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.5.1">
+                                <locus target="115v"></locus>
+                                <title ref="LIT6354MEMiracle"></title>
+                            </msItem>
+                           <msItem xml:id="p1_i2.5.2">
+                               <locus target="116r"></locus>
+                               <title ref="LIT6355MEMiracle"></title>
+                           </msItem>
+                            <msItem xml:id="p1_i2.6">
+                                <locus from="116rb" to="117rb"></locus>
+                                <title ref="LIT6356MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.7">
+                                <locus from="117rb" to="118ra"></locus>
+                                <title></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.8">
+                                <locus from="118ra" to="119rb"></locus>
+                                <title ref="LIT6358MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.9">
+                                <locus from="119rb" to="119vb"></locus>
+                                <title ref="LIT6359MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.10">
+                                <locus from="119vb" to="120ra"></locus>
+                                <title ref="LIT6360MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.11">
+                                <locus from="120ra" to="120rb"></locus>
+                                <title ref="LIT6362MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.12">
+                                <locus from="120rb" to="121va"></locus>
+                                <title ref="LIT6363MEMiracle"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.13">
+                                <locus from="121va" to="121vb"></locus>
+                                <locus from="123ra" to="123vb"></locus>
+                                <title ref="LIT6365MESick"></title>
+                            </msItem>
+                            <msItem xml:id="p1_i2.14">
+                                <locus from="124ra" to="127ra"></locus>
+                                <title></title>
+                                <note>No. 14 corresponds to No. 17 in <bibl><ptr target="bm:Turaiev1905Eustathii"/><citedRange unit="page">167-177</citedRange></bibl> and in
```

Comment:
you do not need the note if you manage to find the correct reference for the title, just as we did for the Miracles of Jesus
if the miracle is not present on BM it must be created

Reply:
If you cannot find the ID and think a work is missing but is needed, before creating it you should create an issue here https://github.com/BetaMasaheft/Documentation/issues and tag me, dorothea and denis. then we will check and tell you to either go ahead or discover the ID